README.md

```diff
@@ -72,7 +71,6 @@ libsecp256k1 is built using autotools:
 Usage examples
 -----------
   Usage examples can be found in the [examples](examples) directory. To compile them you need to configure with `--enable-examples`.
```

Comment:
Following these updated instructions only produces `example_ecdsa`. Should this also say something like:

The `schnorr` and `ecdh` modules aren't enabled by default, so to compile the corresponding examples you will also need `--enable-module-schnorr` and `--enable-module-ecdh`.

Reply:
Indeed. I think this can be done in a separate PR because seems to be a more general issue. At the moment, we don't explain at all how to enable the modules. Maybe this should go to "build steps".